script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -131,7 +133,7 @@ Looks like I'm in for more taciturn prowling around the city.
 -- Page 283, Offset 17263.
 -- 繁華街から離れた、人気の少ない高架線の下にさしかかった時、アルクェイドはぴたりと足を止めた。
 We've left the city's downtown area behind us,
- and are approaching a deserted railway underpass
+ and we're approaching a deserted railway underpass
```

Comment:
Doesn't read as smoothly to me, `We're` as the subject is already established in the first part of the line.